src/reducers/versions.tsx

```diff
@@ -488,7 +488,9 @@ export const createInternalDiffs = ({
         oldRevision: String(baseVersionId),
         hunks: diff.hunks.map((hunk: ExternalHunk) => ({
           changes: hunk.changes.map((change: ExternalChange) => ({
-            content: change.content,
+            // TODO: remove the call to `replace()` once the API is fixed.
+            // See: https://github.com/mozilla/addons-server/issues/10932
+            content: change.content.replace(/\n$/, ''),
```

Comment:
For paranoia, it might be helpful if you added a test to make sure that new lines in the middle of content do not get removed.

Reply:
:+1: